sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -474,6 +479,54 @@ case class Lag(input: Expression, offset: Expression, default: Expression)
   override val direction = Descending
 }
 
+/**
+ * The NthValue function returns the value of `input` at the `offset`th row of beginning
+ * the window partition (counting from 1). Offsets start at 1. When the value of `input` is null
```

Comment:
We remove `(counting from 1)` as it's duplicated

Reply:
OK